testing/integration/headless/test_endpoints.py

```diff
@@ -133,10 +133,10 @@ def test_api_basic(base_url, uri, expected_keys):
         "/search?q=video",
         "/search/?q=video",
         "/events",
-        "/profile",
-        "/profiles/sheppy",
+        "/signup",
+        "/signin",
+        "/settings",
```

Comment:
So some day this might have to move out of the way and become `/account/settings` or something. Right?

Then, on `/account` that's where we have things like a link to your Settings. A link to your Subscription management, link to FAQ about premium account. etc.

Reply:
@peterbe Probably so, since I think if we use MDN accounts going forward, the account settings page will change or just become part of something bigger. For this PR, I was just concerned with making what we currently have consistent.

Reply:
The most important thing is that we don't make `/en-US/account/?` a permanent redirect to `/en-US/settings` otherwise we can never get back to using that URI for our accounts home page.